src/Client.php

```diff
@@ -85,12 +93,14 @@ public function getSettings(): Settings
     /**
      * {@inheritdoc}
      */
-    public function devPing(?string $value = null): DevPingResponse
+    public function devPing(?DevPingParams $params = null): DevPingResponse
     {
         return DevPingResponse::fromHttpResponse(
             $this->doRequest(new Request('get', 'dev/ping'), [
                 'query' => [
-                    'value' => $value ?? ((string) \time()),
+                    'value' => $params instanceof DevPingParams && is_string($value = $params->getValue())
```

Comment:
```suggestion
                    'value' => $params instanceof DevPingParams && \is_string($value = $params->getValue())
```

here and in all places (with "root namespaced" functions), please

Reply:
Fixed in all places 18f0fa4b96cff68b4389afe9688d9beaa8a39878